client/components/mma/cancel/CancellationContainer.tsx

```diff
@@ -33,9 +31,8 @@ import type {
 } from './productSwitch/productSwitchApi';
 
 const renderSingleProductOrReturnToAccountOverview =
-	(productType: ProductType) =>
-	(data: [MembersDataApiResponse | MembersDataApiItem[]]) => {
-		const mdaResponse = mdapiResponseReader(data);
+	(productType: ProductType) => (data: MembersDataApiResponse) => {
+		const mdaResponse = data;
```

Comment:
Could we name this `mdapiResponse` to keep it consistent?

Reply:
Good call - I've made it consistent through other classes too